fud/fud/stages/xilinx/execution.py

```diff
@@ -35,37 +35,24 @@ def _define_steps(self, input, builder, config):
                 f"delete it. Consider adding `-s {self.name}.save_temps true`."
             )
 
-        @builder.step()
-        def import_libs():
-            """Import optional libraries"""
-            try:
-                from fud.stages.xilinx import fud_pynq_script
-
-                self.pynq_script = fud_pynq_script
-            except ImportError:
-                raise errors.RemoteLibsNotInstalled
+        orig_dir = os.getcwd()
+        # Create a temporary directory (used in configure()) with an xrt.ini
+        # file that redirects the runtime log to a file so that we can control
+        # how it's printed. This is hacky, but it's the only way to do it.
+        # (The `xrt.ini`file we currently have in `fud/bitstream` is not used here.)
+        new_dir = FreshDir() if save_temps else TmpDir()
 
         @builder.step()
-        def run(xclbin: SourceType.Path) -> SourceType.String:
-            """Run the xclbin with datafile"""
-
-            if data_path is None:
-                raise errors.MissingDynamicConfiguration("fpga.data")
-            abs_data_path = Path(data_path).resolve()
-            abs_xclbin_path = xclbin.resolve()
+        def configure():
+            """Create config files based on fud arguments"""
 
-            # Create a temporary directory with an xrt.ini file that redirects
-            # the runtime log to a file so that we can control how it's printed.
-            # This is hacky, but it's the only way to do it. (The `xrt.ini`
-            # file we currently have in `fud/bitstream` is not used here.)
-            new_dir = FreshDir() if save_temps else TmpDir()
             os.chdir(new_dir.name)
 
-            xrt_output_logname = "output.log"
+            self.xrt_output_logname = "output.log"
```

Comment:
Made this a field because it is used in [run](https://github.com/cucapra/calyx/pull/1126/files#diff-196539fd2380246a4e4264feccfbf1ce4ae309268226c510dbcd370082f787d1R118) but logically felt like it's definition should be in the config step. Could put this out into the `define_steps` part